schnell/space_detector.py

```diff
@@ -0,0 +1,369 @@
+import numpy as np
+import scipy.integrate as integr
+from .detector import Detector, LISAlikeDetector
+
+class LISADetector2(LISAlikeDetector):
```

Comment:
So the difference wrt the default LISA is adding the GCN?

Reply:
The default LISA was just there as a safeguard, in case the LISAlike superclass didn't work. It should be deleted now (and LISADetector2 renamed to LISADetector)
Same for ALIA